packages/docs/src/pages/qwikcity/layout/nested.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Qwikcity - Nested Layout
+---
+
+# Qwikcity - Nested Layout
+
+At times it is desirable to nest layouts into each other.
+
+The developer could extract all of these into `<Header>`, `<Footer>`, and `<Menu>` components, but that is error-prone and repetitive. Instead, we can use layouts to automatically reuse common parts.
```

Comment:
This seems to be the same content as in `named.mdx`.
I do not see any content being related to nested layouts.

... but I think this is still work in progress. :-)

Reply:
can you have a second look?